src/Clock.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  today: Date;
+};
+
+export class Clock extends React.Component<Props> {
+  state: State = {
+    today: new Date(),
+  };
+
+  timerId = 0;
```

Comment:
```suggestion
  intervalId = 0;
```
I think that `timerId` isn't aptly named.